src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5070,9 +5070,6 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_VarInvocationLvalueReserved" xml:space="preserve">
     <value>The syntax 'var (...)' as an lvalue is reserved.</value>
   </data>
-  <data name="ERR_ExpressionVariableInConstructorOrFieldInitializer" xml:space="preserve">
```

Comment:
>ERR_ExpressionVariableInConstructorOrFieldInitializer [](start = 14, length = 53)

I assume the new behavior will be gated behind a language version. Is that correct?